src/webgpu/api/operation/rendering/stencil.spec.ts

```diff
@@ -319,3 +320,64 @@ g.test('stencil_fail_operation')
     ];
     t.runStencilStateTest(baseState, testStates, 2, _expectedColor);
   });
+
+g.test('stencil_disabled')
+  .desc(
+    `
+  Test that for all combinations, the last triangle drawn is the one visible if the stencil is
+  disabled. We need to check against three triangles because the stencil test may modify results.
+  `
+  )
+  .fn(async t => {
+    const depthSpencilFormat: GPUTextureFormat = 'depth24plus-stencil8';
+
+    const stencilState = {
+      compare: 'always',
+      failOp: 'keep',
+      passOp: 'keep',
+    } as const;
+
+    const baseState = {
+      format: depthSpencilFormat,
+      depthWriteEnabled: false,
+      depthCompare: 'always',
+      stencilFront: stencilState,
+      stencilBack: stencilState,
+      stencilReadMask: 0xff,
+      stencilWriteMask: 0xff,
+    } as const;
+
+    const state = {
+      format: depthSpencilFormat,
+      depthWriteEnabled: false,
+      depthCompare: 'always',
+      stencilFront: stencilState,
+      stencilBack: stencilState,
+      stencilReadMask: 0xff,
+      stencilWriteMask: 0xff,
+    } as const;
```

Comment:
This comment may not matter given the comment above. But leaving it here anyway.

Since this test is specifically testing the "disabled" state it should use all of the defaults, i.e. just not enabling depth/stencil testing, which is the "canonical" way of disabling it. The WebIDL defaulting should have the same results as what you've written.

```suggestion
    const baseState = { format: depthSpencilFormat };
    const state = { format: depthSpencilFormat };
```

I'm guessing the Dawn test is very old and was written before we had defaulting support in webgpu_cpp.h.

Reply:
Thank you for informing me of it. Let me try to apply it to the next patchset.

Reply:
I applied it to the last patchset.